src/user/configure-metadata.md

```diff
@@ -5086,6 +5086,20 @@ View".
 > appears after view B in alphabetical order, analytics may fail, as the
 > view with dependencies will not be dropped in the correct order.
 
+## Manage Analytics Table Hooks { #maintenance_analytics_table_hooks } 
```

Comment:
Should add "Analytics table hooks" to the list of things you can configure under _Configure metadata_ / _About the Maintenance app_. This list is already missing:
- External map layers
- SQL views
- Locales

We should fix this oversight by adding these as well, not add to the oversight.

Note that "Program metadata: tracked entity, tracked entity attribute and relationship type" is in the list of things you can configure but is not a section under _Configure metadata_. This is quirky but OK because it has its own section _Configure programs in the Maintenance app_.

Reply:
External map layers was there, but the others were indeed missing.  I will add them.